docs/ironic-endpoint-keepalived-configuration.md

```diff
@@ -95,20 +35,20 @@ tree resources/keepalived-docker/
 ```
 
 It is assumed that the docker image is uploaded in some registry and the image
-URL is used with `ironic-keepalived-config/image_patch.yaml` to replace the
-default image URL with the correct URL through kustomization.
+URL is used with `ironic-deployment/keepalived/keepalived_patch.yaml` to replace
+the default image URL with the correct URL through kustomization.
 
 **Important Note**
```

Comment:
I forgot to add, **Important Note** part is already included in [deploying.md](https://github.com/metal3-io/baremetal-operator/blob/master/docs/deploying.md), do we need to keep this here as well?

Reply:
I don't see any harm. I would rather keep it here specially since the variables are expected for this container.